Remotion/Core/Collections.Caching.UnitTests/InvalidationTokenBasedCacheDecoratorTest.cs

```diff
@@ -93,21 +94,20 @@ public void GetOrCreateValue_AfterTokenWasInvalidated_RefreshesRevision_ReturnsV
     [Test]
     public void GetOrCreateValue_RetriesClearUntilInvalidationTokenIsCurrent ()
     {
-      var cacheStub = MockRepository.GenerateStub<ICache<object, string>>();
+      var cacheStub = new Mock<ICache<object, string>>();
       var invalidationToken = InvalidationToken.Create();
-      var decorator = new InvalidationTokenBasedCacheDecorator<object, string> (cacheStub, invalidationToken);
+      var decorator = new InvalidationTokenBasedCacheDecorator<object, string> (cacheStub.Object, invalidationToken);
       var key = new object();
       var count = 0;
-      cacheStub.Stub (_ => _.Clear()).WhenCalled (
-          mi =>
-          {
+      cacheStub.Setup (_ => _.Clear()).Callback (
+          () => {
```

Comment:
Sorry, 2nd formatting issue: formultiline anonymous methods, the opening brace is always on a separate line, alined with the parameter list.
```suggestion
          () =>
          {
```